lib/aba.rb

```diff
@@ -41,6 +60,30 @@ def add_transaction(transaction)
     @transactions << transaction
   end
 
+  def transactions
+    @transactions
+  end
+
+  def transactions_valid?
+    transaction_errors = {}
+    @transactions.each_with_index do |transaction, i|
+      transaction_errors[i] = transaction.errors unless transaction.valid?
+    end
+    @transaction_errors = transaction_errors if transaction_errors.length
+    @transaction_errors.empty?
+  end
+
+  # Not updated until transactions_valid? is called - simulates Validations.errors
+  def transaction_errors
+    @transaction_errors
+  end
+
+  # Updates @transaction_errors - simulates Validations.get_errors
+  def get_transaction_errors
```

Comment:
I was just trying to behave like the errors in the Validation module. I'd rather it was smarter as well. I'll look into it.

I was thinking about having `valid?` simply validate the whole thing.

`aba.transactions` is actually not accessible at the moment, so would have to sort that out as well.